### PR TITLE
removed task draw and doit-graph; added awscli and docker-compose

### PR DESCRIPTION
**Pull request by sidler-mozilla**

### CODE REVIEW

File: automation_requirements.txt [hunk 1]
```diff
@@ -7,4 +7,6 @@ tox==3.13.2
 black==19.3b0
 ruamel.yaml==0.15.97
 locustio==0.11.0
-pyparsing==2.4.0
\ No newline at end of file
+pyparsing==2.4.0
+awscli==1.16.247
```
Comment (bowlofstew) [inline automation_requirements.txt]: This should also go into a separate pull request.

**Comment by marty331** [comment]
> Unable to run `doit test` as the following error occurs -
> ERROR: Could not install packages due to an EnvironmentError: [Errno 13] Permission denied: 'RECORD'
> Consider using the `--user` option or check the permissions.
> 
> TaskFailed - taskid:venv
> Command failed: '[ -f "src/app_requirements.txt" ] && /Users/mballard/PycharmProjects/subhub/venv/bin/python3.7 -m pip install -r "src/app_requirements.txt"' returned 1

This is resolved.
